client/components/circles.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Circles = ({ rating }) => { // eslint-disable-line
```

Comment:
You might be able to reduce the lines of code here by dynamically generating the right number of span tags with .map on a helper array that contains the same number of elements as the rating.

Reply:
That would probably be nicer.  I ran into an issue between how React and CSS talk to each other or it would have been way easier.  This was a last minute hack when I got frustrated.